src/objects/DataImportBatch__c.object

```diff
@@ -55,6 +55,16 @@
     <enableSearch>true</enableSearch>
     <enableSharing>true</enableSharing>
     <enableStreamingApi>true</enableStreamingApi>
+    <fields>
+        <fullName>Active_Fields__c</fullName>
+        <description>Represents the data model of active fields for a NPSP_DataImportBatch__c record. Not intended for direct customer use and should not be edited directly.</description>
+        <externalId>false</externalId>
+        <label>Active Fields</label>
+        <length>30000</length>
```

Comment:
Is there a reason to cap this below 32,768?

Reply:
Nope! Where does the standard of 32,768 come from?

Reply:
That's the default limit on a long text area, though it looks like the maximum is actually higher: https://help.salesforce.com/articleView?id=000267856&language=en_US&type=1

I was just wondering why you picked something other than the default. We should also consider whether we'll want to go directly for the maximum as field length may be something we can't alter after packaging.